.ruby-version

```diff
@@ -1 +1 @@
-ruby-3.0.0
+ruby-2.7.2
```

Comment:
TIL that you can't set ruby version in `.ruby-version` file without the patch version, eg `ruby-2.7`.
Because of this my local dev environment doesn't work because my ruby version is `2.7.4`.

In ideal case we should use only `Gemfile` for specifying the version like this `ruby "~> 2.7.0"`

Here is great article about it: https://andycroll.com/ruby/use-loose-ruby-versioning-in-your-gemfile/